### ISSUE TITLE
Hide project instead of deleting them

### ISSUE THREAD

**Issue by CryptoVaraan** [leokhoa/laragon#59]
Menu | Tools | Delete project would be better when it just does hide the directories that do not really contain any projects. Currently all directories under the server root are assumed being projects.

**Comment by leokhoa** [comment]
@CryptoVaraan : How does Laragon know which directories should be hid?

**Comment by CryptoVaraan** [comment]
If you implement Menu | Tools | Hide then the user can indicate what sub-directories of root do not contain a project (black listing), it may be even better to do the opposite and indicate what directories do contain projects (white listing). It does require some administration, like putting a .laragon project file in the root or the indicated directories. I feel deleting projects/directories is dangerous and should not be the responsibility of Laragon.  - Do i really need to say i like Laragon a lot?  I DO, how could i not! Also give i t a star, would give more if possible :)

**Comment by leokhoa** [comment]
Thanks for your star :)

I can't figure out why a directory intentionally put in DocumentRoot but not a project. Implement the feature is easy but will make Laragon GUI complicated. I'm thinking of putting a .laragonignore file inside directories you want to hide.

**Comment by CryptoVaraan** [comment]
The directory structure i have locally is a 1:1 copy of what is at my hosting server. The root there has 15 directories that do NOT contain projects. They are in my case (.cagefs, .cl selector, .cpanel, .cphorde, .htpasswords, .softaculous, .trash, etc, mail, ocartdata, perl5, public_ftp, srbackups, ssl, tmp). The root for my primary domain is served from public_html directory. Every additional hosted site is served from an additional directory under the root (so the same level as the html_root for the primary).

I'm using Filezilla to sync my local tree with what is on the server and have synchronized browsing enabled. Using PHPStorm i have project in the directories that contain actual sites. Laragon is creating virtual hosts and all magic on the irrelevant directories, so i am looking for a way to hide these.

I assume that there will be other with the same "problem"? Your suggestion for a .laragonignore file is a workable approach. Adding a graphical interface for this is probably overkill and too much work to implement.

**Comment by leokhoa** [comment]
@CryptoVaraan : I see your point.
I've released Laragon 4.0.9 which may help you. You can add **IgnoreList** directive in laragon.ini (**Menu > Laragon > laragon.ini**):
```
[pref]
IgnoreList=mail|ocartdata|perl5|public_ftp|srbackups|ssl|tmp
```
Laragon will ignore them in:
1. hosts file
2. Virtual Hosts
3. SSL certificates (in case you enable SSL)

**Note:** You may need to click "Menu > Apache > site-enabled > Delete all auto..." to clean them.

**Comment by CryptoVaraan** [comment]
Thank you for the effort so far! Can you please also remove the ignored ones from Menu | host_root ?  Now it is still possible to click a link to a non-project location.

**Comment by leokhoa** [comment]
@CryptoVaraan : I've fixed it in Laragon 4.0.10

**Comment by CryptoVaraan** [comment]
GREAT! Now it works perfect. I still would like to see **Menu | Tools | Delete project** being removed, since it should not be the responsibility of Laragon to manage deletion of entire projects... but that may be only my way of thinking  :smile:
